nixos/modules/services/misc/jackline.nix

```diff
@@ -0,0 +1,82 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.jackline;
+in {
+  options.services.jackline = {
+    enable = mkOption {
+      description = ''
+        Whether to enable jackline, a minimalistic secure XMPP client in OCaml.
+
+        By doing so, jackline will be run in a screen session. One can attach
+        this session via <literal>sudo -u jackline screen -x jackline-screen</literal>.
+      '';
+      type = types.bool;
+      default = false;
+      example = true;
+    };
```

Comment:
```suggestion
    enable = mkEnableOption ''
        Whether to enable jackline, a minimalistic secure XMPP client in OCaml.
        
        By doing so, jackline will be run in a screen session. One can attach
        this session via <literal>sudo -u jackline screen -x jackline-screen</literal>.
      '';
```

Reply:
Your suggested change would result in a description of "Whether to enable Whether to enable jackline, a minimalistic secure XMPP client in OCaml. […] </literal>.."

https://github.com/NixOS/nixpkgs/blob/a26de40d124cbe084aef15e92746bd26ef7b4e51/lib/options.nix#L93-L100

I have chosen not to use `mkEnableOption`, as it is imho not really usable for multi-line descriptions.